packages/hooks/src/useWalletsList.ts

```diff
@@ -0,0 +1,35 @@
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const useWalletList = () => {
+    const { client } = useStore();
+    const { accounts, loginid, is_crypto } = client;
+    const { data, ...reset } = useFetch('authorize', {
+        payload: { authorize: accounts[loginid || ''].token },
+        options: { enabled: Boolean(loginid) },
+    });
+
+    // @ts-expect-error Need to update @deriv/api-types to fix the TS error
+    const wallets = data?.authorize?.account_list?.filter(account => account.account_category === 'wallet');
+    const modified_wallets = wallets?.map(wallet => ({
+        ...wallet,
+        balance: 1000,
+        landing_company_shortcode: wallet.landing_company_name,
```

Comment:
@thisyahlen-deriv Let's not do this, There is no reason for mapping server response to something else, By doing so we lose the documentation we get from TypeScript and also we create miscommunication between FE and BE over time.

Reply:
I agree. @mahdiyeh-deriv what do you think about this ?